apps/admin_app/lib/admin_app_web/templates/product/form.html.eex

```diff
@@ -58,7 +58,12 @@
         </div>
       </div>
     </div>
-    <div class="form-group row ">
+
+    <div data-toggle="collapse" data-target="#advanced_section" style="margin-bottom:1rem;padding:0.5rem;border:1px solid #ddd;cursor:pointer" >
```

Comment:
you can add the styles in css.

Reply:
Needed style specific for this element and it won't be reused, so inlined here